app/setup.py

```diff
@@ -324,6 +327,22 @@ def setup_submitter(application):
         raise Exception("Unknown EQ_SUBMISSION_BACKEND")
 
 
+def setup_publisher(application):
+    if application.config["EQ_PUBLISHER_BACKEND"] == "pubsub":
+        topic_id = application.config.get("EQ_PUB_SUB_TOPIC_ID")
+
+        if not topic_id:
+            raise Exception("Setting EQ_PUB_SUB_TOPIC_ID Missing")
+
+        application.eq["publisher"] = PubSub(topic_id)
+
+    elif application.config["EQ_PUBLISHER_BACKEND"] == "log":
```

Comment:
Unsure why we have a log publisher, what is it used for?

Reply:
It is used for development same as Log Submitter. We don't use the actual GCP Pub/Sub as we decided not to use an emulator.